src/libraries/System.Runtime.Serialization.BinaryFormat/src/System/Runtime/Serialization/BinaryFormat/ObjectNullMultiple256Record.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+
+namespace System.Runtime.Serialization.BinaryFormat;
+
+/// <summary>
+/// Represents multiple (less than 256) <see langword="null" />.
+/// </summary>
+/// <remarks>
+/// ObjectNullMultiple256 records are described in <see href="https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-nrbf/24ae84a0-781f-45bf-a1ac-49f6a65af5dd">[MS-NRBF] 2.5.6</see>.
+/// </remarks>
+internal sealed class ObjectNullMultiple256Record : NullsRecord
+{
+    private ObjectNullMultiple256Record(byte count) => NullCount = count;
+
+    public override RecordType RecordType => RecordType.ObjectNullMultiple256;
+
+    internal override int NullCount { get; }
+
+    internal static ObjectNullMultiple256Record Decode(BinaryReader reader)
+        => new(reader.ReadByte());
```

Comment:
This code permits 0, and the spec permits 0.  Hopefully there's a test for 0.

Reply:
Good catch, I am going to disallow 0. (and I don't care that spec allows for it, such input does not make any sense and BF is not producing it and the spec was written based on BF implementation, not the other way around)